src/figuresClasses.ts

```diff
@@ -1,19 +1,75 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a >= (b + c) || b >= (a + c) || c >= (b + a)
+    || (a <= 0 || b <= 0 || c <= 0)
+    ) {
+      throw new Error('Error');
```

Comment:
Check the 3-rd point in the [checklist](https://github.com/mate-academy/ts_figures_classes/blob/master/checklist.md). Change it in all places

Reply:
Fixed